apps/api/src/util/redis-indexing.ts

```diff
@@ -0,0 +1,12 @@
+import type { IoC } from '@intake24/api/ioc';
+
+// Add a value to the redisSetService if needed IndexRebuilding
+export const addToRedisIndexingKeyCache = async (value: string, { cache }: Pick<IoC, 'cache'>) => {
```

Comment:
Shouldn't this rather be method in cache? Seems kind of an anti-pattern to pull dep out of IoC and run specific thing for cache.